src/pages/workspace/card/WorkspaceCardVBAWithECardView.tsx

```diff
@@ -53,30 +52,24 @@ function WorkspaceCardVBAWithECardView(props) {
 
     return (
         <Section
-            title={props.translate('workspace.card.headerWithEcard')}
+            title={translate('workspace.card.headerWithEcard')}
             icon={Illustrations.CreditCardsNew}
             menuItems={menuItems}
             isCentralPane
         >
             <View style={[styles.mv3]}>
-                <Text>{props.translate('workspace.card.VBAWithECardCopy')}</Text>
+                <Text>{translate('workspace.card.VBAWithECardCopy')}</Text>
             </View>
 
             <View style={[styles.mv3]}>
                 <UnorderedList
-                    items={[
-                        props.translate('workspace.card.benefit1'),
-                        props.translate('workspace.card.benefit2'),
-                        props.translate('workspace.card.benefit3'),
-                        props.translate('workspace.card.benefit4'),
-                    ]}
+                    items={[translate('workspace.card.benefit1'), translate('workspace.card.benefit2'), translate('workspace.card.benefit3'), translate('workspace.card.benefit4')]}
```

Comment:
As we have done in other files, can we move this to a separate const.

Reply:
You are right, missed that one, thank you!